Vagrantfile

```diff
@@ -11,10 +11,15 @@ SCRIPT
 
 DATA_DIR = ENV["RASTER_VISION_DATA_DIR"]
 if DATA_DIR.nil?
-  puts "You must set the environment variable: RASTER_VISION_DATA_DIR"
+  puts "ERROR: You must set the environment variable: RASTER_VISION_DATA_DIR"
```

Comment:
The changes in this file are already on the develop branch along with a bug fix at https://github.com/azavea/raster-vision/blob/develop/Vagrantfile#L34